src/fetchtastic/download/client_app.py

```
@@ -0,0 +1,1128 @@
+        if (
+            asset.size is not None
+            and self.file_operations.get_file_size(target_path) != asset.size
+        ):
+            return False
+        if not self.verify(target_path):
+            return False
+        if target_path.lower().endswith(".zip") and not self._is_zip_intact(
+            target_path
+        ):
+            return False
+        return True
```

Ruff (0.15.12)

[warning] Return the negated condition directly

Inline condition

(SIM103)